docs/images/SearchCommandSequenceDiagram.svg

```diff
@@ -0,0 +1,141 @@
 y2="627.7383"/><text fill="#000000" font-family="sans-serif" font-size="9.75" lengthAdjust="spacingAndGlyphs" textLength="21.75" x="792" y="624.0959">order</text><polygon fill="#A80036" points="956.25,647.502,963.75,650.502,956.25,653.502,959.25,650.502" style="stroke: #A80036; stroke-width: 0.75;"/><line style="stroke: #A80036; stroke-width: 0.75;" x1="417.75" x2="960.75" y1="650.502" y2="650.502"/><text fill="#000000" font-family="sans-serif" font-size="9.75" lengthAdjust="spacingAndGlyphs" textLength="21.75" x="680.625" y="646.8596">order</text><rect fill="#3333C4" height="23.707" style="stroke: #A80036; stroke-width: 1.125;" width="100.5" x="965.25" y="633.7383"/><text fill="#FFFFFF" font-family="sans-serif" font-size="10.5" lengthAdjust="spacingAndGlyphs" textLength="90" x="970.5" y="649.8882">s:SearchCommand</text><polygon fill="#A80036" points="426,679.459,418.5,682.459,426,685.459,423,682.459" style="stroke: #A80036; stroke-width: 0.75;"/><line style="stroke: #A80036; stroke-width: 0.75; stroke-dasharray: 2.0,2.0;" x1="421.5" x2="1014.75" y1="682.459" y2="682.459"/><text fill="#000000" font-family="sans-serif" font-size="9.75" lengthAdjust="spacingAndGlyphs" textLength="5.25" x="714" y="678.8167">s</text><polygon fill="#A80036" points="291.75,707.4727,284.25,710.4727,291.75,713.4727,288.75,710.4727" style="stroke: #A80036; stroke-width: 0.75;"/><line style="stroke: #A80036; stroke-width: 0.75; stroke-dasharray: 2.0,2.0;" x1="287.25" x2="413.25" y1="710.4727" y2="710.4727"/><text fill="#000000" font-family="sans-serif" font-size="9.75" lengthAdjust="spacingAndGlyphs" textLength="5.25" x="346.125" y="706.8303">s</text><polygon fill="#A80036" points="134.25,730.2363,126.75,733.2363,134.25,736.2363,131.25,733.2363" style="stroke: #A80036; stroke-width: 0.75;"/><line style="stroke: #A80036; stroke-width: 0.75; stroke-dasharray: 2.0,2.0;" x1="129.75" x2="279" y1="733.2363" y2="733.2363"/><text fill="#000000" font-family="sans-serif" font-size="9.75" lengthAdjust="spacingAndGlyphs" textLength="5.25" x="200.25" y="729.594">s</text><rect fill="#FFFFFF" height="32.0273" style="stroke: #000000; stroke-width: 1.5;" width="1436.25" x="72" y="739.2363"/><polygon fill="#EEEEEE" points="72,739.2363,118.5,739.2363,118.5,745.2363,111,752.7363,72,752.7363,72,739.2363" style="stroke: #000000; stroke-width: 1.5;"/><text fill="#000000" font-family="sans-serif" font-size="9.75" font-weight="bold" lengthAdjust="spacingAndGlyphs" textLength="12.75" x="81.75" y="750.8577">ref</text><text fill="#000000" font-family="sans-serif" font-size="9.75" lengthAdjust="spacingAndGlyphs" textLength="237" x="671.625" y="765.8577">Execution of the Search Command Sequence Diagram</text><polygon fill="#A80036" points="10.5,776.5137,3,779.5137,10.5,782.5137,7.5,779.5137" style="stroke: #A80036; stroke-width: 0.75;"/><line style="stroke: #A80036; stroke-width: 0.75; stroke-dasharray: 2.0,2.0;" x1="6" x2="121.5" y1="779.5137" y2="779.5137"/><!--MD5=[14422ba3ca44a1e3273b057733c2f2d1]
```

Comment:
Might want to remove the 'ref'.
As the diagram is quite large, you might want to split it up.